accepted/future-releases/constructor-tearoffs/feature-specification.md

```diff
@@ -295,54 +306,92 @@ Example:
 ```dart
 T id<T>(T value) => value;
 int Function(int) idInt = id; // Implicitly instantiated tear-off.
-
+// and
 typedef IntList = List<int>;
 Type intList = IntList;
 ```
 
-We will introduce syntax allowing you to explicitly instantiate a function tear-off and a type literal for a generic class. The former for consistency with constructor tear-offs, the latter to introduce in-line types without needing a `typedef`, like we did for in-line function types originally. And we do both now because they share the same grammar productions.
+This feature introduces syntax to explicitly instantiate a generic function object or tear-off and a type literal for a generic type. The former for consistency with constructor tear-offs, the latter to introduce in-line types without needing a `typedef`, like we did for in-line function types originally. We introduce both now because they share the same grammar productions.
 
 Example:
 
 ```dart
 T id<T>(T value) => value;
-var idInt = id<int>; // Explicitly instantiated tear-off, saves on writing function types.
+var idInt = id<int>; // Explicitly instantiated tear-off, saves on function types.
 // and
 Type intList = List<int>; // In-line instantiated type literal.
 ```
 
-These grammar changes allows *type parameters* without following parenthesized arguments in places where we previously did not allow them. For example, this means that `<typeArguments>` becomes a *selector* by itself, not just followed by arguments.
+These grammar changes allow *type parameters* without following parenthesized arguments in places where we previously did not allow such. This means that `<typeArguments>` becomes a *selector* by itself, not just when followed by arguments.
```

Comment:
How about "did not allow that." 
The "them" can refer to both "type parameters" or "following parenthesized arguments" (both plural nouns earlier in the sentence). It's more obvious that it's type parameters, because of the recurrence of "allow", but it's not unambiguous. I want to refer to allowing the combination of "type parameters without a following argument list".

Reply:
Sure, "did not allow that" works for me.